Source/FakeItEasy/Core/DefaultFakeWrapperConfigurer.cs

```diff
@@ -6,21 +6,27 @@ namespace FakeItEasy.Core
     /// Handles configuring of fake objects to delegate all their calls to a wrapped instance.
     /// </summary>
     internal class DefaultFakeWrapperConfigurer
-        : IFakeWrapperConfigurer
     {
+        public DefaultFakeWrapperConfigurer(object wrappedObject)
+        {
+            this.WrappedObject = wrappedObject;
+        }
+
+        public object WrappedObject { get; private set; }
+
+        public ISelfInitializingFakeRecorder Recorder { get; set; }

```

Comment:
The introduction of a mutable field always sets off an alarm bell in my head but I'm tying myself in knots trying to explore a way to avoid it. I don't really understand the collaboration of the various types well enough.

If you believe this is the best solution then I'm happy to roll with it.